code/modules/events/supermatter_surge.dm

```diff
@@ -13,7 +13,7 @@
 	var/power = 2000
 
 /datum/round_event/supermatter_surge/setup()
-	power = rand(400,8000)
+	power = rand(1000,100000)
 
 /datum/round_event/supermatter_surge/announce()
 	if(power > 800 || prob(round(power/8)))
```

Comment:
1) Both these conditions are redundant as the power will never be under 800 and the prob will never fail as the minimum is 1000/8 so prob(125).
2) The class number can be ridiculously high now. Anywhere from class 3 to class 201

Reply:
1. removed the condition so it always announces,
2. yes